### PR TITLE
Remove semicolon after function declaration

### LINKED ISSUE #8417: Issue with "Graceful asynchronous programming with Promises": (semicolon after function definition)
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises

#### What information was incorrect, unhelpful, or incomplete?
semicolon at the end of codeblock for function "timeoutPromise" not needed. It's only a definition and no code is being executed here. Same mistake in custom-promise2.html under the same headline

#### Specific section or headline?
Rejecting a custom promise

#### What did you expect to see?
I expected no semicolon at the end of a definition of a function.

#### Did you test this? If so, how?
tested twice. 
once just copied the code and did it without a semicolon. works fine
second time I edited custom-promise2.html and it also works without semicolon.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/javascript/asynchronous/promises`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/asynchronous/promises/index.html
* Last commit: https://github.com/mdn/content/commit/b04ce27c5af69e3583dd6021f71da2bca780228b
* Document last modified: 2021-07-12T05:41:51.000Z

</details>

### PR DESCRIPTION
**Pull request by AnilSeervi**
- remove `seosummary` and `standard-table` class
- add semicolon after function expression
- fixes #8417

### CODE REVIEW

File: files/en-us/learn/javascript/asynchronous/promises/index.html [hunk 1]
```diff
@@ -279,7 +279,7 @@ <h3 id="Chaining_the_blocks_together">Chaining the blocks together</h3>
 
 <h2 id="Promise_terminology_recap">Promise terminology recap</h2>
 
-<p>There was a lot to cover in the above section, so let's go back over it quickly to give you a <a href="/en-US/docs/Learn/JavaScript/Asynchronous/Promises#promise_terminology_recap">short guide that you can bookmark</a> and use to refresh your memory in the future. You should also go over the above section again a few more times to make sure these concepts stick.</p>
+<p>There was a lot to cover in the above section, so let's go back over it quickly to give you a short guide that you can bookmark and use to refresh your memory in the future. You should also go over the above section again a few more times to make sure these concepts stick.</p>
```
Comment (teoli2003) [inline files/en-us/learn/javascript/asynchronous/promises/index.html:282]: Why did you remove this link?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Just one question.

File: files/en-us/learn/javascript/asynchronous/promises/index.html (hunk 1 above)
Comment (AnilSeervi) [reply files/en-us/learn/javascript/asynchronous/promises/index.html:282]: The link is the current page on which it is and it's also linking to the same section on which it is, so I removed it.

Comment (teoli2003) [reply files/en-us/learn/javascript/asynchronous/promises/index.html:282]:
```suggestion
<p>There was a lot to cover in the above section, so let's go back over it quickly to give you a <a href="#promise_terminology_recap">short guide that you can bookmark</a> and use to refresh your memory in the future. You should also go over the above section again a few more times to make sure these concepts stick.</p>
```

What about linking to the fragment?

Comment (AnilSeervi) [reply files/en-us/learn/javascript/asynchronous/promises/index.html:282]: I guess the link is not that necessary here.

Comment (teoli2003) [reply files/en-us/learn/javascript/asynchronous/promises/index.html:282]: Oh yes, I missed this. You're right.

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks!
